app/controllers/accept_account_invitations_controller.rb

```diff
@@ -1,18 +1,24 @@
 # frozen_string_literal: true
 
 class AcceptAccountInvitationsController < ApplicationController
-  before_action :authenticate_user!
+  def show
+    return if user_signed_in? && the_invitation
 
-  def show; end
+    session[:token] = ps[:token]
+    redirect_to new_user_session_url(email: the_invitation.email)
+  end
 
   def update
     received_invitation.update!(accepted_at: Time.current)
-    redirect_to account_path(received_invitation.account_id)
```

Comment:
I agree that it still was not working, but it was the correct line. Revert it back? Why removed?

Reply:
Ok, let's keep